### ISSUE TITLE
Should not trigger on byte strings

### ISSUE THREAD

**Issue by nyuszika7h** [MichaelKim0407/flake8-use-fstring#22]
The following code triggers an FS001 error:
```py
a = 42
b"y=%d" % a
```
However, using an f-string with a byte string is a syntax error:
```pycon
>>> bf"y={a}"
```
```pytb
  File "<stdin>", line 1
    bf"y={a}"
      ^^^^^^^
SyntaxError: invalid syntax
```

(In this particular case `f"y={a}".encode("utf-8")` would work, but that's longer and doesn't always work correctly if there are byte literals in my string.)

**Comment by MichaelKim0407** [comment]
OK I can take a look. Should be relatively easy to change.

Can you please check what greedy level you are using? If you are using level 2 then it would report an error anyway.

**Comment by nyuszika7h** [comment]
I'm using whatever is the default.
